docker-wrappers/SPRAS/README.md

```diff
@@ -38,27 +38,97 @@ Or to temporarily override your system's default during the build, prepend your
 DOCKER_DEFAULT_PLATFORM=linux/amd64
 ```
 
-For example, to build reedcompbio/spras:v0.1.0 on Apple Silicon as a linux/amd64 container, you'd run:
+For example, to build reedcompbio/spras:v0.2.0 on Apple Silicon as a linux/amd64 container, you'd run:
 ```
-DOCKER_DEFAULT_PLATFORM=linux/amd64 docker build -t reedcompbio/spras:v0.1.0 -f docker-wrappers/SPRAS/Dockerfile .
+DOCKER_DEFAULT_PLATFORM=linux/amd64 docker build -t reedcompbio/spras:v0.2.0 -f docker-wrappers/SPRAS/Dockerfile .
 ```
 
-## Testing
+### Converting Docker Images to Apptainer/Singularity Images
 
-The folder `docker-wrappers/SPRAS` also contains several files that can be used to test this container on HTCondor. To test the `spras` container
-in this environment, first login to an HTCondor Access Point (AP). Then, from the AP clone this repo:
+It may be necessary in some cases to create an Apptainer image for SPRAS, especially if you intend to run your workflow using distributed systems like HTCondor. Apptainer (formerly known as Singularity) uses image files with `.sif` extensions. Assuming you have Apptainer installed, you can create your own sif image from an already-built Docker image with the following command:
+```bash
+apptainer build <new image name>.sif docker://<name of container on DockerHub>
+```
 
+For example, creating an Apptainer image for the `v0.2.0` SPRAS image might look like:
+```bash
+apptainer build spras-v0.2.0.sif docker://reedcompbio/spras:v0.2.0
 ```
+
+After running this command, a new file called `spras-v0.2.0` will exist in the directory where the command was run.
```

Comment:
Is the file called `spras-v0.2.0.sif`?

Reply:
Should be resolved. Thanks for catching it!